snuba/datasets/replays_processor.py

```diff
@@ -225,5 +235,5 @@ def process_message(
             return InsertBatch([processed], None)
         except Exception as e:
             metrics.increment("consumer_error")
-            capture_exception(e)
+            logger.exception(e)
```

Comment:
`logger.exception("Replay could not be consumed.")` or something similar.  Traceback is pulled implicitly.

Reply:
right.